pages/table/resizable-columns-percentage.page.tsx

```diff
@@ -0,0 +1,233 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Would it make sense to modify the existing resizable-columns page with an ability to define column widths using some settings?

Reply:
makes sense, I was worried about making it overly complex, but I think it works better overall having it in one place